pvr.hts/addon.xml.in

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.hts"
-  version="4.0.9"
+  version="4.0.10"
   name="Tvheadend HTSP Client"
   provider-name="Adam Sutton, Sam Stenvall, Lars Op den Kamp, Kai Sommerfeld">
   <requires>
     <c-pluff version="0.1"/>
-    <import addon="xbmc.pvr" version="5.2.2"/>
+    <import addon="kodi.global.main" version="@ADDON_GLOBAL_VERSION_MAIN@"/>
+    <import addon="kodi.instance.pvr" version="@ADDON_INSTANCE_VERSION_PVR@"/>
```

Comment:
@ksooo is the use of something like this OK, or better for you to change version always by hand?